configurator/parsers.py

```diff
@@ -8,11 +10,19 @@ class ParseError(Exception):
     """
 
 
+if sys.version_info >= (3, 11):
+    # stdlib
+    _toml_mod = 'tomllib'
+else:
+    _toml_mod = 'tomli'
```

Comment:
I'd prefer the binary stuff to be here, can the `tomli` one use a `BytesIO` or something in  wrapper function here?

Reply:
I assuming you mean wrapping the tomli/tomllib with our own callable which has the interface that the `Parsers` expects, rather than passing along the load function directly. I'd considered that, but it would still require some refactoring because of the way [`__import__` is used below](https://github.com/simplistix/configurator/blob/v3.1.0/configurator/parsers.py#L26) - this only works for top-level modules like `toml` and it won't work for a submodule like `configurator._toml`  :(